### PR TITLE
chore: improve lighthouse performance

### PR DESCRIPTION
**Pull request by JeevaRamu0104**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
Added alt attribute to image tag to improve the FCP and added preconnect and preload to the link tags
image
image

## Checklist

- [ ] I ran `npm run re:build`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/entryPoints/HyperSwitchEntry.res [hunk 1]
```diff
@@ -107,7 +107,7 @@ module HyperSwitchEntryComponent = {
       }
 
       None
-    }, (name, email, Window.env.mixpanelToken))
+    }, featureFlagDetails.mixpanel)
```
Comment (PritishBudhiraja) [inline src/entryPoints/HyperSwitchEntry.res:110]: why this?

File: public/hyperswitch/index.html [hunk 2]
```diff
@@ -1,108 +1,133 @@
...
-        display: flex;
-        flex-direction: column;
-      }
 
-      #app {
-        display: flex;
-        flex-direction: column;
-        flex: 1;
-      }
+<head>
```
Comment (PritishBudhiraja) [inline public/hyperswitch/index.html:4]: is it just formatting or something changed?

File: src/entryPoints/HyperSwitchEntry.res (hunk 1 above)
Comment (JeevaRamu0104) [reply src/entryPoints/HyperSwitchEntry.res]: updated

File: public/hyperswitch/index.html (hunk 2 above)
Comment (JeevaRamu0104) [reply public/hyperswitch/index.html:4]: formatting as well as some changes are done

**Review by PritishBudhiraja — DISMISSED** [review]

File: public/hyperswitch/index.html [hunk 3]
```diff
@@ -1,108 +1,132 @@
...
-</html>
+      });
+    }
+  </script>
+</head>
+
+<body>
+  <!-- SDK Loader Manager (noscript) -->
+  <script defer type="module" src="/wasm/euclid.js"></script>
+  <script defer src="/module.js"></script>
```
Comment (Riddhiagrawal001) [inline public/hyperswitch/index.html:127]: For both of this can use async

**Review by sagarnaikjuspay — DISMISSED** [review]

File: public/hyperswitch/index.html (hunk 3 above)
Comment (Riddhiagrawal001) [reply public/hyperswitch/index.html:127]: Can we also move these scripts below `app.js`

File: public/hyperswitch/index.html [hunk 4]
```diff
@@ -1,108 +1,132 @@
...
+    }
 
-      .linkAnnotation {
-        width: 20% !important;
-      }
+    @font-face {
+      font-family: 'IBM Plex Sans';
+      src: url('https://fonts.gstatic.com/s/ibmplexsans/v8/zYXgKVElMYYaJe8bpLHnCwCzB.woff2') format('woff2');
+      font-display: swap;
+    }
```
Comment (Riddhiagrawal001) [inline public/hyperswitch/index.html:41]: I dont think we need this as we are passing in the href of those google fonts

File: public/hyperswitch/index.html (hunk 3 above)
Comment (JeevaRamu0104) [reply public/hyperswitch/index.html:109]: not required

Comment (PritishBudhiraja) [reply public/hyperswitch/index.html:109]: Let it be defer only as async may cause issues. In defer let the script fully load and then execution start

File: public/hyperswitch/index.html (hunk 4 above)
Comment (PritishBudhiraja) [reply public/hyperswitch/index.html]: @JeevaRamu0104 can we remove one of the font as I guess we are only using the Inter or IBM only one. Why to fetch 2 fonts unnecessarily.

File: public/hyperswitch/index.html [hunk 5]
```diff
@@ -1,108 +1,114 @@
...
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    as="style" />
+  <link rel="preload"
+    href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&family=IBM+Plex+Sans:wght@300;400;500;600;700&display=swap"
+    as="style" />
 
-      .list-item:hover {
-        background-color: rgba(0, 0, 0, 0.1);
-      }
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
```
Comment (PritishBudhiraja) [inline public/hyperswitch/index.html:18]: We are using Inter we can remove the rest two link tags

File: public/hyperswitch/index.html (hunk 3 above)
Comment (Riddhiagrawal001) [reply public/hyperswitch/index.html:109]: These are scripts which are not needed from the start and also does not need DOM to render hence we can use async

File: public/hyperswitch/index.html [hunk 6]
```diff
@@ -1,108 +1,107 @@
...
-        flex: 1;
-      }
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta charset="UTF-8" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preload"
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+    as="style" />
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
```
Comment (PritishBudhiraja) [inline public/hyperswitch/index.html:14]: we can remove this. twice not required

**Review by PritishBudhiraja — APPROVED** [review]

**Pull request merged**